lib/cog_api/http/base.ex

```diff
@@ -27,13 +27,7 @@ defmodule CogApi.HTTP.Base do
 
   def delete(%Endpoint{}=endpoint, resource) do
     rescue_econnrefused(fn ->
-      response = HTTPotion.delete(make_url(endpoint, resource), headers: make_headers(endpoint))
-      case response_type(response) do
-        :ok ->
-          :ok
-        :error ->
-          {:error, Poison.decode!(response.body)}
-      end
+      HTTPotion.delete(make_url(endpoint, resource), headers: make_headers(endpoint))
```

Comment:
That's being handled by `format_response` by the calling function, which brings it up to spec with `get`, `post` and `patch`